Taxi/Taxi/Presenter/CalendarView.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  CalendarView.swift
+//  Taxi
+//
+//  Created by sanghyo on 2022/06/09.
+//
+
+import SwiftUI
+
+struct CalendarView: View {
+    @State private var currentDate = Date()
+    @State private var selectedDate = Date()
+    @State private var currentMonth = 0
+    private let today = Date()
+    private let taxiParties: [TaxiParty] = TaxiPartyMockData.mockData
+    private let action: () -> Void
+    private let calendarType: CalendarType
+    let days = ["일", "월", "화", "수", "목", "금", "토"]
```

Comment:
```suggestion
    private let days = ["일", "월", "화", "수", "목", "금", "토"]
```

이 친구도 접근제한자 설정 가능해보여욥